### DISCUSSION TITLE
This repo include a copy of Mono?

### DISCUSSION THREAD

**Discussion by ghost** [dotnet/runtime#99038] [General] [answered]
Please pardon my ignorant. I'm not a .NET expert nor I take a close track of .NET development. I found Mono on the submitted issues and commit messages. It seems this version of Mono is for Android and WASM. Please correct me if I'm wrong.

I also wonder if the included Mono is the same as the one on https://github.com/mono/mono? I don't think they are the same. The one included here is more active developed and the patches are not backported to the one on https://github.com/mono/mono. The one on https://github.com/mono/mono is pretty much dead. No one to review commits. No one to reply to submitted issues. The state of it is too bad that someone decided to fork it on https://github.com/mono/mono/issues/20931.

**Comment by huoyaoyuan** [comment, marked as answer]
The mono code in this repository is effectively a *fork*. The mono/mono repo contains **.NET Framework compatible Mono**, while this repo contains **.NET Core compatible Mono**. The .NET Framework compatible Mono is as inactive as .NET Framework itself.

**Reply by ghost** [reply]
Is there any plan to spun this fork into a separate repo? I think many people will interested in a standalone **.NET Core compatible Mono**.

**Reply by davidfowl** [reply]
No, that’s the opposite of what we want to do. The runtimes share code and we want them to share more and more over time.

**Reply by danmoseley** [reply]
Btw it sounds like you're new here @iahung2 ? Welcome.
